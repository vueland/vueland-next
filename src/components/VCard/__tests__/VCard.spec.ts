import { mount, VueWrapper } from '@vue/test-utils'
import { VCard } from '../index'

describe('VCard', () => {
  let mountFunction: (options?: any) => VueWrapper<any>

  beforeEach(() => {
    mountFunction = (options = {}) => mount(VCard, { ...options })
  })

  it('should mount component and match snapshot', () => {
    const cmp = mountFunction()

    expect(cmp.html()).toMatchSnapshot()
  })
})
