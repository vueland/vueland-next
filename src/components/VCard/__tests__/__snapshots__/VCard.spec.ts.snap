// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VCard should mount component and match snapshot 1`] = `
<div class="v-card"
     style="width: 350px;"
>
</div>
`;
